CONTRIBUTING.md

```diff
@@ -1,11 +1,77 @@
-# Contributing
+# Overview
 
-**Technical Steering Committee (TSC)**
+This project aims to be governed in a transparent, accessible way for the benefit of the community. All participation in this project is open and not bound to corporate affilation. Participants all our bound the ASWF Code of Conduct.
+
+# Project roles
+
+## Contributor
+
+The contributor role is the starting role for anyone participating in the project and wishing to contribute code.
+
+# Process for becoming a contributor
```

Comment:
Is signing the CLA also a requirement?

Reply:
Yes - added this in there now. Good catch!